setup.py

```diff
@@ -67,6 +67,8 @@
         'Operating System :: OS Independent',
     ],
     packages=find_packages(),
+    package_data={'tests': '*.xml'},
+    data_files=[('', ['LICENSE'])],
```

Comment:
I haven't personally used conda. Does msal in conda still ship with an `PKG-INFO` file which would then contain a line as `License: MIT`, just like what pip did? If so, I would suggest to also rollback this line too.

Reply:
Conda Forge is quite keen on actually having an actual license_file: https://conda-forge.org/docs/maintainer/adding_pkgs.html#packaging-the-license-manually